deployment/arm/deploy-global.sh

```diff
@@ -68,18 +47,20 @@ then
     exit 1
 fi
 
-scope="$businessUnit-$appName-$env-gbl"
-regionScope="$businessUnit-$appName-$env"
-resourceGroupName="rg-$scope"
-frontDoorName="fd-$scope"
-cosmosdbName="db-$scope"
+# include common script to populate shared variables
+source common-script.sh
+
+# set global resource group name
```

Comment:
Any reason not to move this into the `common-script.sh` also?

Reply:
the resource group name is unique to the deployment: region/global, thus I kept it in each file respectively. But I can move it to common and have globalResourceGroupName and regionalResourceGroupName..let me know what would you prefer?

Reply:
True - nevermind 👍